manual/dna-manual.Rnw

```diff
@@ -983,6 +983,8 @@ dna_gui(infile = dna_sample())
 @
 If these commands can be executed correctly, you are ready to use both \dna\ and \rdna.
 
+Note that when updating \rdna\ you will also need the newest release of \dna\ for both programs to work together properly. You can use the command \code{dna\_downloadJar()} to download the last release version of \dna.
```

Comment:
Replace "newest" by "latest".
Move second sentence to separate line.
Replace "last" by "latest".